lib/Db/Card.php

```diff
@@ -106,6 +107,7 @@ public function __construct() {
 		$this->addType('lastModified', 'integer');
 		$this->addType('createdAt', 'integer');
 		$this->addType('archived', 'boolean');
+		$this->addType('done', 'boolean');
```

Comment:
```suggestion
		$this->addType('completed', 'datetime');
```
See the reasoning for this in https://github.com/nextcloud/deck/pull/4137#issuecomment-1521730474. Of course, all other occurrences of `done` would need to be adjusted as well.

Reply:
Yes, i think that makes a lot of sense. Not only for the dav backend but also for future features.

Added as a todo entry to the first post: 
> - [ ]  Move from boolean column to datetime to store that exact timestamp when a card was marked as done